### ISSUE TITLE
What are some planned updates to the sample theme?

### ISSUE THREAD

**Issue by paaljoachim** [copyblogger/genesis-sample#8]
Hey

I am curious as to what will be added/updated to the sample theme?

Have a great day!

**Comment by GaryJones** [comment]
At the very least, it will get some accessibility improvements.

**Comment by bgardner** [comment]
Gary is right about the accessibility. The jury is still out of the Sample theme will undergo some cosmetic changes as well, but more than likely it won't.

**Comment by bgardner** [comment]
And if it did, I was considering moving it towards a mobile-first approach like this: http://themedemo.me/mobile-first/

@GaryJones -- thoughts on that?

**Comment by scarinessreported** [comment]
@bgardner I dont think the sample theme should have any cosmetic changes at the expense of accessibility and mobile first. 

The only cosmetic change from a localisation point of view is maybe changing to Noto font. I know most people can do it themselves but maybe it falls into the accessibility category also

**Comment by RRWD** [comment]
@scarinessreported @bgardner I'm planning to request some changes in the colour contrast, so they comply with WCAG 2

**Comment by GaryJones** [comment]
@bgardner You _know_ I'm going to be +1 to mobile-first order of styles for Sample theme :smiley: 

Also, +1 for child theme accessibility fixes including colour contrast changes, and the [better way of loading Google Web Fonts](https://github.com/cdils/in-the-beginning/blob/master/functions.php#L65) to account for languages that contain characters not present in some fonts.

In short, I think Sample should contain every best practice we know of, including RTL and editor style sheets. If someone is opinionated enough to say they don't like an aspect of it (i.e. mobile-first) then they are free to make their own starter theme. For everyone else, having a theme that already contains all the goodies is a good thing - more efficient work flow and education about aspects of theme development they didn't even know existed.

**Comment by bgardner** [comment]
All sounds good, and I agree @GaryJones. Once we have all of the Genesis 2.2 issues resolved and it's getting prepped for release, we can move to the Sample theme. Ideally, we'd release them both at the same time.
